common/src/main/java/com/wynntils/functions/CharacterFunctions.java

```diff
@@ -188,6 +189,38 @@ public List<Class<? extends Model>> getModelDependencies() {
         }
     }
 
+    public static class MoneyFunction extends Function<Integer> {
+        @Override
+        public Integer getValue(String argument) {
+            int emeralds = ContainerUtils.getEmeraldCountInContainer(McUtils.inventoryMenu());
+            return emeralds;
+        }
+    }
+
+    public static class LeFunction extends Function<Integer> {
```

Comment:
I would rename these to be LiquiedEmeraldFunction (same for Eb and Em). You can override `getAliases` and add "le", "eb" and "em" as an alias respectively.